alpha/apps/kaltura/lib/kUploadTokenMgr.php

```diff
@@ -117,6 +117,24 @@ public function getUploadToken()
 	 */
 	public function uploadFileToToken($fileData, $resume = false, $resumeAt = -1)
 	{
+		if (!$this->_uploadToken->getFileName())
```

Comment:
there is a function for that on line 148, no need to so it twice
$this->updateFileName($fileData);